data/framework-laptop12.tablet

```diff
@@ -0,0 +1,17 @@
+[Device]
+Name=Framework Laptop 12
+Class=ISDV4
+# VID belongs to Ilitek, the touch IC vendor
+# PID is reserved for this Framework Laptop 12
+DeviceMatch=i2c|222a|5539
```

Comment:
Just for reference, needs at least 2.12 with https://github.com/linuxwacom/libwacom/commit/767ae7ecaaf47213bce967dfce115ed6a6f9118f

Ubuntu 24.04 only has 2.10, so they need to use colon separated match:


```
DeviceMatch=i2c:222a:5539
```